python/calico/felix/masq.py

```diff
@@ -63,6 +63,9 @@ def on_ipam_pool_updated(self, pool_id, pool):
                 _log.info("IPAM pool deleted: %s", pool_id)
                 del self.pools_by_id[pool_id]
             else:
+                if ":" in pool["cidr"]:
```

Comment:
Python felix never supported IPv6 pool here but they were filtered out by not watching for them in the first place.